Developer Samples/LibraryLogEnabler/LibLogEnabler/LoggingEnabler.cs

```diff
@@ -1,14 +1,13 @@
 ﻿using System;
 using System.Web;
-//LibLog References
-using TraceLogger.Logging;
-using TraceLogger.Logging.LogProviders;
```

Comment:
Was this change required due to the version change, or some other reason?

Reply:
nuget upgrade automatically updated everything from TraceLogger to LibLogEnabler, so trace logger is no more, it seems.